### PR TITLE
fix: update home page card shape and bottom menu theme

### PR DESCRIPTION
**Pull request by britannio**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here: https://github.com/WorldHealthOrganization/app/issues/750
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?
- Closes https://github.com/WorldHealthOrganization/app/issues/750
- Sets the card tile border-radius to 16 on the home page
- Add white background to buttons at the bottom of the home page
- Changes the arrow color to #C9CDD6 on the arrows at the bottom of the home page

## How did you test the change?

Screenshot_20200401-192713

### CODE REVIEW

File: client/flutter/lib/pages/home_page.dart [hunk 1]
```diff
@@ -182,40 +182,86 @@ class _HomePageState extends State<HomePage> {
...
+                ),
               ),
-              Divider(),
-              ListTile(
-                leading: Icon(Icons.settings, color: Color(0xffCA6B35)),
-                title: Text(S.of(context).homePagePageSliverListSettings, style: TextStyle(color: Color(0xffCA6B35), fontWeight: FontWeight.w600, fontSize: 20),),
-                trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () => Navigator.of(context)
-                    .push(MaterialPageRoute(builder: (c) => SettingsPage())),
+              Divider(height: 1),
```
Comment (jamesblasco) [inline client/flutter/lib/pages/home_page.dart:217]: For making the divider thicker you need to add thickness:1. Height only defines the space height but not the width of the line.

Comment (britannio) [reply client/flutter/lib/pages/home_page.dart:217]: My intention was to remove the extra padding around the divider not change the thickness.

Comment (jamesblasco) [reply client/flutter/lib/pages/home_page.dart:217]: Then I think height: 0 would better as the thickness is smaller than 1 by default. But it is okey like this 👍

Comment (britannio) [reply client/flutter/lib/pages/home_page.dart:217]: I've had inconsistent behavior with height: 0 in other apps but never mind.

Comment (jamesblasco) [reply client/flutter/lib/pages/home_page.dart:217]: Nice to know, thank you

**Pull request merged**